README.md

```diff
@@ -210,6 +210,18 @@ This will copy the `hosts.yaml` configuration file from
 that it is picked up by Pennyworth automatically. This makes it easy to
 distribute a common configuration between several systems and users.
 
+Pennyworth automatically cleans up the hosts after running the tests using
+snapper snapshots. The snapshot which the system is rolled back to is specified
+by the `base_snapshot_id` of the host entries. Before rolling back a new
+snapshot named `pennyworth` is created which can be used for debugging test
+failures.
+
+In some cases (e.g. while working on a test) the automatic rollback might not be
+desired. It can easily be disabled temporarily by setting the `SKIP_CLEANUP`
+environment variable, e.g.
+
+    SKIP_CLEANUP=true rspec
```

Comment:
The use case I had in mind (and I already ran into) is that you're working on a test and need to run it a few times until you got it right. There it sometimes makes sense to skip the rollback/reboot  step, but it's not really an attribute of the test but rather of the test run. That's why I made it depend on the env variable instead of the code.